tpl/liquid/css/_admin.less

```diff
@@ -0,0 +1,61 @@
+.dokuwiki div.ui-admin {
```

Comment:
Instead of committing all this junk, you could `cp -r /opt/bitnami/dokuwiki/lib/tpl/dokuwiki /liquid/tpl/liquid` in `Dockerfile` and then overwrite the modified files.